services/fxa/resources/tags.yml

```diff
@@ -0,0 +1,15 @@
+default:
+  cost-center: 1440
+  project-name: subhub
+  project-desc: payment subscription REST api for customers
```

Comment:
Should we correct the casing of `api` to match the preceding acronym?

Reply:
yes